packages/desktop-gui/src/project-nav/browsers.jsx

```diff
@@ -30,16 +31,16 @@ export default class Browsers extends Component {
   }
 
   _closeBrowserBtn () {
-    if (this.props.project.browserState === 'opened') {
-      return (
-        <li className='close-browser'>
-          <button className='btn btn-xs btn-danger' onClick={this._closeBrowser.bind(this)}>
-            <i className='fa fa-fw fa-times'></i>
-            Stop
-          </button>
-        </li>
-      )
-    }
+    if (!this.props.project.isBrowserState(Project.BROWSER_OPEN)) return null
```

Comment:
Nice! I like this short-circuit. It reduces overall cyclomatic complexity and, more importantly, makes the main case of the function the proper responsibility; i.e. the main purpose of the function was not to return null; this was simply an edge case.